core/src/main/scala/doric/syntax/ArrayColumns.scala

```diff
@@ -525,7 +525,8 @@ private[syntax] trait ArrayColumns {
       * @note
       * if `start` == 0 an exception will be thrown
       * @group Array Type
-      * @see [[org.apache.spark.sql.functions.slice(x:org\.apache\.spark\.sql\.Column,start:org\.apache\.spark\.sql\.Column,length* org.apache.spark.sql.functions.slice]]
+      * @see org.apache.spark.sql.functions.slice
```

Comment:
This is one of the URL's I couldn't get working. If there is any advice here I would much appreciate it!

Reply:
@eruizalo you are the master of links, anything that can be done?

Reply:
we could cry together...

let me take a look later today if I can test something, but I do not have much hope solving this right now

anyway I think this is not a stopper, so, in the worst scenario we could merge this without links and try out again later

Reply:
@eruizalo thanks for having a look into this and also fixing the issues with Spark 2.4, it's much appreciated! 🎉 

Is there anything else you are looking into or shall I mark this PR as ready for review? In the meantime I will continue to try and see if I can get these other links to work as well.

Reply:
Hi @Amalicia ! I thought I wrote you about this! I'm really sorry 🤕 

I tried some tricks but nothing worked... I seized the opportunity to rebase, resolve conflicts and fix some new errors I got. I will continue looking for a solution for this if my mental sanity allows me to do it.

I think this PR is ready for review and merge.

Thank you very much for your help!

Reply:
Absolutely no worries at all! Thanks again for all the help on this and hopefully you haven't been _too_ destroyed mentally 😉